src/main/java/io/frictionlessdata/tableschema/field/BooleanField.java

```diff
@@ -80,16 +79,6 @@ public String parseFormat(String value, Map<String, Object> options) {
     }
 
     public static Field fromJson (String json) {
-        BooleanField field = (BooleanField)Field.fromJson(json);
```

Comment:
Thanks @iSnow, I had to look at this a couple times too =]  - SInce it's parsing those fields as String arrays [trueValues, falseValues], jackson will automatically map them in the BooeanField class.
The 'parseValue` function should still validate the truthy/falsey values -  
```
    @Override
    public Boolean parseValue(String value, String format, Map<String, Object> options)
            throws InvalidCastException, ConstraintsException {
        if (null != options) {
            if (options.containsKey("trueValues")) {
                trueValues = new ArrayList<>((Collection) options.get("trueValues"));
            }
            if (options.containsKey("falseValues")) {
                falseValues = new ArrayList<>((Collection) options.get("falseValues"));
            }
        }

        if (trueValues.contains(value.toLowerCase())){
            return true;

        }else if (falseValues.contains(value.toLowerCase())){
            return false;

        }else{
            throw new TypeInferringException();
        }
    }
```

Reply:
Thanks. Unfortunately, it seems there is no test for it, and even the master test-data suite from frictionless doesn't seem to have a Schema that uses non-standard true/false values.

As this is not something in the scope of this bounty, I am merging your changes and pushing to master and will craft a test later. 

Thanks for your work, I am in awe about your speed switching this to Jackson!

Reply:
Thank you @iSnow !!!